### DISCUSSION TITLE
Revocation certificate

### DISCUSSION THREAD

**Discussion by ghost** [kklash/mnemonikey#43] [General]
A user might want to keep a revocation certificate separate to their normal mnemonikey, somewhere where they or a trusted third party can revoke it.

Is it possible to generate a revocation certificate in BIP39?

I haven't had a look at the code yet, thought I'd ask first.

Cheers

**Comment by kklash** [comment]
Hey there, thanks for opening a discussion. I think this is outside the scope of mnemonikey, as you can generate revocation certificates easily using `gpg --gen-revoke <key_id>` as long as you have the primary key in your keychain. Subkeys can be revoked inside the `gpg --edit <key_id>` menu.
